router/interviews.js

```diff
@@ -12,7 +12,7 @@ const {
 const interviewsRouter = Router();
 
 interviewsRouter.get('/', getInterviews);
-interviewsRouter.get('/interview/:id', getInterviewsById);
+interviewsRouter.get('/id/:id', getInterviewsById);
```

Comment:
I think it's better to get rid of the `/id`

```suggestion
interviewsRouter.get('/:id', getInterviewsById);
```